src/couch_file.erl

```diff
@@ -524,6 +528,214 @@ handle_info({'EXIT', _, Reason}, Fd) ->
     {stop, Reason, Fd}.
 
 
+find_last_header(Fd, FileSize) ->
+    find_header_in_chunks(Fd, start_pos(FileSize), 1).
+
+start_pos(FileSize) when FileSize rem ?SIZE_BLOCK =:= 0 ->
+    FileSize;
```

Comment:
Why not just rely on integer math below as per SOP on these sorts of calculations?